packages/fx-core/src/component/driver/deploy/azure/azureStorageDeployDriver.ts

```diff
@@ -191,10 +176,7 @@ export class AzureStorageDeployDriverImpl extends AzureDeployImpl {
     return (blob.properties.contentLength ?? -1) > 0;
   }
 
-  createProgressBar(ui?: UserInteraction): IProgressHandler | undefined {
-    return ui?.createProgressBar(
-      `Deploying ${this.workingDirectory ?? ""} to Azure Storage Service`,
-      this.progressNames.length
-    );
+  createProgressBar() {
```

Comment:
Is this function invoked only 1 time?

Reply:
Yes, I will rename it to 'updateProgressbar' for clarity.